packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -67,7 +83,7 @@ class MediaQueryData {
       boldText = window.accessibilityFeatures.boldText,
       alwaysUse24HourFormat = window.alwaysUse24HourFormat;
 
-  /// The size of the media in logical pixel (e.g, the size of the screen).
+  /// The size of the media in logical pixels (e.g, the size of the screen).
```

Comment:
`The size of the media in logical pixels or points (e.g, the size of the screen).`

To make it friendlier for mobile developers' lexicon

Reply:
I don't think that will make it friendlier since it's not clear how "points" which typically would be a font dimension, map to logical pixels.

Reply:
https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/Displays/Displays.html
https://developer.android.com/guide/topics/resources/more-resources#Dimension

'points' and 'dp' are just terms of art on iOS/Android.